content/en/security/application_security/threats/library_configuration.md

```diff
@@ -0,0 +1,116 @@
+  tag: "Documentation"
+  text: "Out-of-the-Box Application Security Management Rules"
+- link: "/security/application_security/add-user-info/"
+  tag: "Documentation"
+  text: "Adding user information to traces"
+- link: "/security/application_security/troubleshooting"
+  tag: "Documentation"
+  text: "Troubleshooting ASM"
+- link: "/security/application_security/how-appsec-works/"
+  tag: "Documentation"
+  text: "How Application Security Management Works in Datadog"
+---
+
+
+## Configuring a client IP header
+
+ASM automatically attempts to resolve `http.client_ip` from several well-known headers, such as `X-Forwarded-For`. If you use a custom header for this field, or want to bypass the resolution algorithm, set the `DD_TRACE_CLIENT_IP_HEADER` environment variable and the library only checks the specified header for the client IP.
+
+## Track authenticated bad actors
+
+Many critical attacks are performed by authenticated users who can access your most sensitive endpoints. To identify bad actors that are generating suspicious security activity, add user information to traces by instrumenting your services with the standardized user tags. You can add custom tags to your root span, or use instrumentation functions. Read [Tracking User Activity][1] for more information.
+
+## Exclude specific parameters from triggering detections
+
+There may be a time when an ASM signal, or a suspicious request, is a false positive. For example, ASM repeatedly detects
+the same suspicious request and a signal is generated, but the signal has been reviewed and is not a threat.
+
+You can add an entry to the passlist, which ignore events from a rule, to eliminate noisy signal patterns and focus on legitimately suspicious requests.
+
+To add a passlist entry, do one of the following:
+
+- Click on a signal in [ASM Signals][4] and click the **Add Entry** link next to the **Add to passlist** suggested action. This method automatically adds an entry for the targeted service.
+- Navigate to [Passlist Configuration][5] and manually configure a new passlist entry based on your own criteria.
+
+**Note**: Requests (traces) that match a passlist entry are not billed.
+
+## Data security considerations
+
+The data that you collect with Datadog can contain sensitive information that you want to filter out, obfuscate, scrub, filter, modify, or just not collect. Additionally, it may contain synthetic traffic that might cause your threat detection be inaccurate, or cause Datadog to not accurately indicate the security of your services.
+
+By default, ASM collects information from suspicious requests to help you understand why the request was flagged as suspicious. Before sending the data, ASM scans it for patterns and keywords that indicate that the data is sensitive. If the data is deemed sensitive, it is replaced with a `<redacted>` flag, so you observe that although the request was suspicious, the request data could not be collected because of data security concerns.
```

Comment:
```suggestion
By default, ASM collects information from suspicious requests to help you understand why the request was flagged as suspicious. Before sending the data, ASM scans it for patterns and keywords that indicate that the data is sensitive. If the data is deemed sensitive, it is replaced with a `<redacted>` flag. This enables you to observe that although the request was suspicious, the request data was not collected because of data security concerns.
```